### PR TITLE
Adding tables as migrations

### PR DESCRIPTION
**Pull request by SaveryIV**
In this project i created migrations of tables and relationships with the foreign keys.

### CODE REVIEW

**Comment by SaveryIV** [comment]
I dont know why but when i try to execute the migrations i receive an error

**Comment by SaveryIV** [comment]
image

File: db/migrate/20231101182340_create_post.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreatePost < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
```
Comment (Strangeal) [inline db/migrate/20231101182340_create_post.rb]: - [ ] 🚀 Impressive work so far in your implementation! 👏 However, I'd like to bring up a minor detail in your code that can further enhance it and ensure it aligns perfectly with all the project requirements.

   - 👉 In your `post` table, I noticed that you're referencing the `user` column on the `user` table as a `foreign_key`. According to the project's Entity-Relationship Diagram (ERD) and the explicit requirements, the `posts` table should be linked to the `users` table via the `author_id` column, not the `user` column.

**📜 Here's a quick reminder from the project requirements:**
   Screenshot 2023-11-01 at 19 33 38
   
 Happy coding! 😊🚀👨‍💻

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @SaveryIV, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Well structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @SaveryIV ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Implemented requested changes ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**